virttest/utils_net.py

```diff
@@ -4322,3 +4323,30 @@ def delete_ovs_bridge(ovs_bridge_name, session=None, ignore_status=False):
     return utils_misc.cmd_status_output(tmux_cmd, shell=True, verbose=True,
                                         ignore_status=ignore_status,
                                         session=session)
+
+
+def get_iface_info(mac, session=None):
+    """
+    Get host/inside-vm info of certain interface with given mac address
+
+    :param mac: mac address of given interface
+    :param session: session of given vm, default to None
+    :return: dict-type info of interface, None if not get any
+    """
+
```

Comment:
Docstring linters detect this and (rightfully) complain that there shouldn't be empty lines after function docstrings.